modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -727,6 +727,7 @@ The following information is available in the JSON format:
 
 | database
 | The database name the command is executed on.
+This field depends on the security event.
```

Comment:
Is this trying to say that this field will not be populated for all events, it depends on the context, because that is true. A failed login would not have a database attached to it because if you do not supply correct credentials you don't get on a database.

Reply:
Yes, I am trying to say that. It's was just difficult to define for which events it will be populated and for which not. How about: 
```suggestion
This field depends on the RBAC event.
```